README.md

```diff
@@ -10,7 +10,16 @@ npm install -g lightning-cli
 
 ## Usage
 
+#### Immediate scan
 ```
-lightning <url>
+lightning scan <url>
 ```
 _url_: valid url of the website you want to be analyzed.
+
+#### Schedule a recurring scan
+```
+lightning schedule <url> --interval --start
+```
+_url_: valid url of the website you want to be analyzed.
+_--interval_: A time interval in milliseconds. Defaults to one day
```

Comment:
should we add that there's a min value for interval, even if it might be temporary?